src/CompanyService.Mappers/Db/DbPositionMapper.cs

```diff
@@ -21,7 +21,7 @@ public DbPosition Map(CreatePositionRequest value, Guid companyId)
                 Id = Guid.NewGuid(),
                 CompanyId = companyId,
                 Name = value.Name,
-                Description = value.Description.Any() ? value.Description : null,
+                Description = value.Description != null && value.Description.Trim().Any() ? value.Description : null,
```

Comment:
value.Description.Trim()

Reply:
It should be like
 Description = value.Description != null && value.Description.Trim().Any() ? value.Description.Trim() : null,

Reply:
done